query-builder/Column.ts

```diff
@@ -50,8 +51,27 @@ export class Column {
       string += " UNSIGNED";
     }
 
-    if (this.defaultValue) {
-      string += ` DEFAULT ${this.defaultValue}`;
+    //process this.defaultValue
```

Comment:
Let's wrap this in a private method, or in an `if` statement like the rest of the handlers. We don't want any of this to execute if `this.defaultValue` is undefined.

Also, add a JSDoc `/** Some doc */` stating where the code is from.